packs/github/actions/create_issue.py

```diff
@@ -0,0 +1,18 @@
+from lib.base import BaseGithubAction
+from github import GithubObject
+from lib.formatters import issue_to_dict
+
+__all__ = [
+    'CreateIssueAction'
+]
+
+
+class CreateIssueAction(BaseGithubAction):
+    def run(self, user, repo, title, description, assignee):
```

Comment:
For any of the optional passed parameters, you'll want to set that parameter to `None`. Otherwise, the runner will get upset when it's expecting that parameter but we're not feeding it through.